src/manager/components/CommentForm/index.js

```diff
@@ -1,7 +1,24 @@
 import React, { Component } from 'react';
 import Textarea from 'react-textarea-autosize';
+import marked from 'marked';
 import style from './style';
```

Comment:
We can't import CSS like this.
Check this: https://github.com/kadirahq/storybook-addon-knobs/pull/26/files#diff-50f961defa6d9bfc44bba2257fc2ff7aR3